jujugui/static/gui/src/app/models/models.js

```diff
@@ -268,6 +268,9 @@ window.yui.add('juju-models', function(Y) {
     }
   }, {
     ATTRS: {
+      supported: {},
```

Comment:
Should this default to false?

Reply:
Good call, added some defaults to all 3.